tensorflow/compiler/xla/service/gpu/ir_emitter_unnested.cc

```diff
@@ -2174,15 +2174,19 @@ void IrEmitterUnnested::EmitPrologueForReduction(
     llvm::Type* element_type =
         llvm_ir::PrimitiveTypeToIrType(reduce_inst->shape().element_type(),
                                        ir_emitter_context_->llvm_module());
-    llvm::AllocaInst* reduction_input_address = Alloca(element_type);
+    llvm::AllocaInst* reduction_input_address =
+        llvm_ir::EmitAllocaAtFunctionEntry(element_type, absl::string_view(),
```

Comment:
same here, as we move multiple `Alloca` to the entry basic block, it's better to name them explicitly. As a matter of fact I think that's why `EmitAllocaAtFunctionEntry` carry a `name` parameter.